### PR TITLE
SceFiber: add error checks

### PR DESCRIPTION
**Pull request by scribam**
~~It helps A.O.T. Wings of Freedom and Grisaia no kajitsu spinout to proceed a bit further.~~

### CODE REVIEW

**Comment by Zangetsu38** [comment]
you can maybe look here
https://github.com/Zangetsu38/Vita3K/commit/83a73b09d6d054f4b8f99031b23e3512e68d9238#diff-b181fd9334f7431a3703ca398d9a46d4caff8e65053b7fa758a8b876f40a59c1R67
code by @sunho

**Comment by scribam** [comment]
Do you know why this code hasn't been pushed in Vita3K?

**Comment by Zangetsu38** [comment]
> 
> 
> Do you know why this code hasn't been pushed in Vita3K?

Have said is probably not totaly sur, ask directly @sunho ^^

**Comment by sunho** [comment]
I wanted to properly investigate those functions with hw testing, but I forgot. (I was not sure whether the arguments are correct) Can you check if " fiber->addrContext = addrContext; fiber->sizeContext = sizeContext;" make difference?

**Comment by scribam** [comment]
I am not sure either about the arguments but it looks like we found the same 🙂 
Adding `fiber->addrContext = addrContext; fiber->sizeContext = sizeContext;` doesn't help the games to process further unfortunately.

**Comment by sunho** [comment]
I see. This function is specifically used in libult lle code. It'd not be used in game code directly. (as long as sane game developer wrote code) So, it's probably fine to stub this wisely.

File: vita3k/modules/SceFiber/SceFiber.h [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 #include <module/module.h>
 
+#define SCE_FIBER_CONTEXT_MINIMUM_SIZE 512
```
Comment (sunho) [inline vita3k/modules/SceFiber/SceFiber.h:22]: What's this?

Comment (scribam) [reply vita3k/modules/SceFiber/SceFiber.h:22]: A constant I used in _sceFiberInitializeImpl and _sceFiberInitializeWithInternalOptionImpl functions to check the size of the context:
```
    if ((sizeContext != 0) && (sizeContext < SCE_FIBER_CONTEXT_MINIMUM_SIZE)) {
        return RET_ERROR(SCE_FIBER_ERROR_RANGE);
    }
```
The size of the context can be equal to 0 or >=SCE_FIBER_CONTEXT_MINIMUM_SIZE

**Review by sunho — APPROVED** [review]

**Pull request merged**